src/main/resources/config/liquibase/changelog/20220731000000_changelog.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.8.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <!-- CleanUp for Fresh Start -->
+    <changeSet author="stefanwaldhauser" id="2022073100000-1" >
+        <preConditions onFail="MARK_RAN"><tableExists tableName="tutor_group_students"/></preConditions>
+        <dropTable cascadeConstraints="true" tableName="tutor_group_students"/>
+    </changeSet>
+    <changeSet author="stefanwaldhauser" id="2022073100000-2" >
+        <preConditions onFail="MARK_RAN"><tableExists tableName="tutor_group"/></preConditions>
+        <dropTable cascadeConstraints="true" tableName="tutor_group"/>
+    </changeSet>
+    <!-- Fresh Start -->
+    <changeSet author="stefanwaldhauser" id="2022073100000-3">
+        <createTable tableName="tutorial_group">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="tutorial_groupPK"/>
+            </column>
+            <column name="title" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="teaching_assistant_id" type="BIGINT"/>
+            <column name="is_online" type="BOOLEAN"/>
+            <column name="course_id" type="BIGINT"/>
+            <column name="additional_information" type="LONGTEXT"/>
+            <column name="language" type="VARCHAR(255)"/>
+            <column name="location" type="LONGTEXT"/>
+            <column name="capacity" type="INT"/>
+        </createTable>
+
+        <createTable tableName="tutorial_group_registration">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="tutorial_groupPK"/>
```

Comment:
Does you relation need a primary key or could the tuple (tutorial_group_id, student_id) act as one?

Reply:
We might support in the future that a student could self register to a tutorial group and be registered by an instructor. So to keep this possibility open, I would rather not make this uniqueness restriction for now.

Reply:
How would that feature influence the primary key in this table? Could you elaborate a little more? 🙂

Reply:
Well if I make tutorial_group_id and student_id the primary key only one registration could exist for one student and one tutorial group if I am not mistaken.

Reply:
This is actually a really good catch, and I overlooked this. As it currently stands, this would cause issues further down the road. See my other comment.